mat64/dense_test.go

```diff
@@ -830,7 +830,7 @@ func TestMul(t *testing.T) {
 		// These probably warrant a better check and failure. They should never happen in the wild though.
 		temp.mat.Data = nil
 		panicked, message := panics(func() { temp.Mul(a, b) })
-		if !panicked || message != "blas: index of c out of range" {
+		if !panicked || message != "blas: insufficient matrix slice length" {
 			t.Error("exected runtime panic for nil data slice")
```

Comment:
While you're here, can you make the condition more informative. Adding the panic message to the error when panicked == true would do it.